tests/installation/partitioning_lvm.pm

```diff
@@ -14,16 +14,24 @@ use base "y2logsstep";
 use testapi;
 
 sub run() {
-    my $self = shift;
+    my $self             = shift;
+    my $file_system_tags = [
+        qw/
+          partitioning-no-root-filesystem partitioning-encrypt-activated-existing
+          partitioning-encrypt-ignored-existing
+          /
+    ];
 
     if (get_var('ENCRYPT_ACTIVATE_EXISTING')) {
-        assert_screen [qw/partitioning-no-root-filesystem partitioning-encrypt-activated-existing/];
+        assert_screen $file_system_tags;
         if (match_has_tag('partitioning-no-root-filesystem')) {
             record_soft_failure 'bsc#989750';
         }
-        else {
-            return;
+        elsif (match_has_tag('partitioning-encrypt-ignored-existing')) {
+            record_soft_failure 'bsc#993247';
```

Comment:
Easy to check: If logs would be gathered you can see that in screenshots and videos. `record_soft_failure` is just incrementing an error counter and outputting the error message in the log file, not more if I am not mistaken.